src/main/java/com/alysaa/geyseradmintools/commands/GatCommand.java

```diff
@@ -15,18 +16,21 @@
     public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
         if (sender instanceof Player) {
             Player player = (Player) sender;
+            if (!player.hasPermission("geyseradmintools.gadmin")) {
+                return false;
+            }
+
             UUID uuid = player.getUniqueId();
             boolean isFloodgatePlayer = CheckJavaOrFloodPlayer.isFloodgatePlayer(uuid);
             if (isFloodgatePlayer) {
-                if (command.getName().equalsIgnoreCase("gadmin") && player.hasPermission("geyseradmintools.gadmin")) {
-                    try {
-                        MainForm.formList(player);
-                    } catch (Exception e) {
-                        e.printStackTrace();
-                    }
+                try {
+                    MainForm.formList(player);
+                    return true;
+                } catch (Exception e) {
+                    e.printStackTrace();
                 }
```

Comment:
checking permission first will likely result in less lines of code being run overall